server/di/di.go

```diff
@@ -120,6 +124,11 @@ func (c *Container) ReplicateExistingClusterService() ReplicateExistingClusterSe
 	return c.replicateExistingClusterService
 }
 
+// WatcherService returns WatcherService.
```

Comment:
hard to understand what this service will do from the name. 
rename to more detailed name. (like `ResourceWatcherService`?)